Makefile

```diff
@@ -38,10 +38,10 @@ lambdas: $(LAMBDAS)
 .PHONY: $(LAMBDAS)
 
 $(LAMBDAS): build/%/bootstrap:
-	GOOS=$(GOOS) GOARCH=$(GOARCH) CGO_ENABLED=$(CGO_ENABLED) $(GOCC) build $(GOFLAGS) -o $@ cmd/lambda/$*/main.go
+	GOOS=$(LAMBDA_GOOS) GOARCH=$(LAMBDA_GOARCH) CGO_ENABLED=$(LAMBDA_CGO_ENABLED) $(LAMBDA_GOCC) build $(LAMBDA_GOFLAGS) -o $@ cmd/lambda/$*/main.go
 
 deploy/.terraform:
-	tofu -chdir=deploy init
+	TF_WORKSPACE= tofu -chdir=deploy init
```

Comment:
Ok so, tofu expects no existing `TF_WORKSPACE` when initializing, but then needs it subsequently.